### DISCUSSION TITLE
customtkinter .configure text_font

### DISCUSSION THREAD

**Discussion by Coding-Mania-Begins-Now** [TomSchimansky/CustomTkinter#135] [Q&A]
I have become quite fond of customtkinter, as it makes it much easier to display widgets efficiently.
However, one thing I have noticed is that you cannot configure `text_font`.
For example, if I were to create a button:
`button = customtkinter.CTkButton(master, text="Hi!", text_font=("Consolas", 25) `
and if I try to configure its `text_font`:
`button.configure(text_font=('Arial', 23))`

I receive this error:
`tkinter.TclError: unknown option "-text_font"`

There should be an option to configure text_font.

**Comment by abhaymakes** [comment]
Hello @Coding-Mania-Begins-Now i've already made an issue for that check it in #266.
